### PR TITLE
Improvement/unify of 3 config files settings and reformat the code to 120 width

### PR DESCRIPTION
**Pull request by Bishop341-B**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module.exports = {
     'env': {
         'browser': true,
-        'es2021': true
+        'es2020': true
```
Comment (loclamor) [inline .eslintrc.js:4]: Why going down to es2020 ?

**Review by loclamor — APPROVED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (Bishop341-B) [reply .eslintrc.js:4]: I think that we do not use any feature of es2021. This way we don't need to enforce a requirement of a most newer browser version, when it really is not needed. We are always in time to raise it when needed, of course.

By the way... precisely i think that some day the minimun browser version declaration must be revised as i think that we are using some newer JS features that these versions do not support.

Comment (loclamor) [reply .eslintrc.js:4]: For firefox you can specify on the store wich versions you support; for chrome i think i necer seen any of this feature. Maybe is it automatic ?

Comment (Bishop341-B) [reply .eslintrc.js:4]: Maybe seems to be this:

https://developer.chrome.com/docs/extensions/mv3/manifest/minimum_chrome_version/

Anyway... anyhow this is made, sometime we have to choose a suitable minimum browsers version for OGI for the JS features it uses. As if it runs in a browser with no support to these features, funny things may happen... XD

**Review by ONeill33 — APPROVED** [review]

**Pull request merged**